### PR TITLE
Upgrade to React 16 and Enzyme 3

### PR DESCRIPTION
**Pull request by jackmallahan**
Per our conversation last week, I ticked this version to v2.0.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pearson-components/elements-sdk",
-  "version": "1.15.5",
+  "version": "2.0.0",
   "description": "Styling and basic JavaScript components for UI development.\"",
```
Comment (antelopeb) [inline package.json]: Go ahead and role this back to 1.15.5. It will get automatically changed during release and we will make sure to tag it properly.

**Comment by NFabrizio** [comment]
Looks good. Let's get this shipped.

**Comment by antelopeb** [comment]
Looks good!! Release to v2.0.0 or let me know if you want me to release it.

**Comment by jackmallahan** [comment]
Thanks @antelopeb, would you mind releasing? I am out of the office today.

**Comment by jackmallahan** [comment]
@antelopeb I'm blocked from merging, can you take care of merging/releasing?

**Comment by antelopeb** [comment]
@jackmallahan -- I merged, but I am getting an error on the test suite:
`Cannot find module 'has-symbols/shams' from 'implementation.js'`

Any suggestions? This is after `npm install` was run as well.

**Pull request merged**